packages/core/src/internal.ts

```diff
@@ -0,0 +1,2 @@
+export * from './index';
+export * from './internal/process-declaration-functions';
```

Comment:
Let's keep the explicit exports
```suggestion
export { processDeclarationFunctions } from './internal/process-declaration-functions';
```